src/contracts/marketplace.ts

```diff
@@ -196,6 +196,13 @@ export class Marketplace implements UpdateableNetwork {
    *
    * @param listingId - the listing id
    * @returns either a direct or auction listing
+   *
+   * @remarks Create and manage auctions in your marketplace.
```

Comment:
this remark doesn't make sense, this is the remark on the `getListing()` function?